wagtailmarkdown/mdx/tables/__init__.py

```diff
@@ -14,8 +14,8 @@
 Copyright 2009 - [Waylan Limberg](http://achinghead.com)
 """
 
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals

```

Comment:
considering we are removing Python 2.7 support, we can get rid of `__future__` imports

https://python-future.org/unicode_literals.html